### DISCUSSION TITLE
Mengganti /mengupdate File Quiz P4DS dan PS dari rmd menjadi HTML sehingga muncul done late

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#540] [Q&A] [answered]
Dari hasil pengerjaan Quiz P4DS dan PS, sebelumnya saya input file rmd nya (sebelum dateline) lalu disarankan utk mengirimkan file HTML nya  yg seharusnya di submit, lalu saya resubmit utk menggantinya dg file HTML. Setelah saya resubmit menjadi done late, apakah ini tidak masalah ? Terimakasih

<img width="868" alt="image" src="https://user-images.githubusercontent.com/123436963/218258114-49f49e80-f294-4403-a30a-e4a1f7d2eba1.png">

<img width="641" alt="image" src="https://user-images.githubusercontent.com/123436963/218258278-1a5028a9-5511-43bd-958f-3d884ab6773a.png">

**Comment by kevbow** [comment, marked as answer]
Dear @Micoalgo,

Hal ini sama sekali tidak masalah. Terima kasih.

**Comment by Micoalgo** [comment]
Terimakasih jawabannya
